src/components/Products.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useFetch } from '../hooks/useFetch';
+import Loading from './LoadingData';
+import Image from './Image';
+import Box from './Box';
+import { Link } from 'react-router-dom';
+
+const Products = () => {
+    const {data: products, loading, error} = useFetch('products/');
+
+
+    if (error){
+      return <p>An error occurred: {error}</p>
+    }
+  
+    if(loading){
+      return <Loading />;
+    }
+  
+    return(
+      <>
+      {products.length &&
```

Comment:
The syntax works fine in React but we can run into problems with it in React Native.

Please use the syntax`{products.length ? <ComponentToRender/> : null}`